chainer/links/caffe/caffe_function.py

```diff
@@ -358,6 +358,12 @@ def _setup_batchnorm(self, layer):
             size, decay=decay, eps=eps, use_gamma=False, use_beta=False)
         func.avg_mean.ravel()[:] = blobs[0].data
         func.avg_var.ravel()[:] = blobs[1].data
+
+        if len(blobs) == 3:
```

Comment:
Can you use `len(blobs) >= 3`?

Reply:
Thanks, I changed the condition accordingly.